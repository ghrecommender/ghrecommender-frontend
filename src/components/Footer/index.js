import React from 'react'

import { Container, Header, Segment, Grid, List, Icon } from 'semantic-ui-react'
import { trackClick } from '../../utils'

const Footer = () => (
  <Segment inverted vertical style={{ padding: '5em 0em' }}>
    <Container>
      <Grid divided inverted>
        <Grid.Row>
          <Grid.Column width={6}>
            <Header inverted as="h4" content="About" />
            <List link inverted>
              <List.Item
                as="a"
                href="mailto:hire@ghrecommender.io"
                onClick={trackClick('Footer - Hire us')}
              >
                <Icon name="envelope outline" /> Hire us
              </List.Item>
              <List.Item
                as="a"
                href="https://github.com/ghrecommender"
                onClick={trackClick('Footer - Github')}
              >
                <Icon name="github" /> GitHub
              </List.Item>
              <List.Item
                as="a"
                href="https://twitter.com/ghrecommender"
                onClick={trackClick('Footer - Twitter')}
              >
                <Icon name="twitter" /> Twitter
              </List.Item>
            </List>
          </Grid.Column>
          <Grid.Column width={6}>
            <p>
              Copyright Â© 2017 <a href="https://github.com/avli">Andrey Lisin</a> and{' '}
              <a href="http://github.com/yurtaev">Egor Yurtaev</a>
            </p>
            <p>The code is licensed under MIT.</p>
          </Grid.Column>
        </Grid.Row>
      </Grid>
    </Container>
  </Segment>
)

export default Footer
