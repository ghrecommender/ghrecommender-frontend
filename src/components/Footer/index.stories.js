import React from 'react'
import { storiesOf } from '@storybook/react'

import Footer from './index'

const stories = storiesOf('Footer', module)
stories.add('<Footer />', () => <Footer />)
