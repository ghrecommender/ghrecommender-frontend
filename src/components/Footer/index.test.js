import React from 'react'
import renderer from 'react-test-renderer'

import Footer from './index'

test('<Footer />', () => {
  const component = renderer.create(<Footer />)
  expect(component.toJSON()).toMatchSnapshot()
})
