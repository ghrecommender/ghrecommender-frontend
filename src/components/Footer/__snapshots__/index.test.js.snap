// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> 1`] = `
<div
  className="ui inverted vertical segment"
  style={
    Object {
      "padding": "5em 0em",
    }
  }
>
  <div
    className="ui container"
  >
    <div
      className="ui inverted divided grid"
    >
      <div
        className="row"
      >
        <div
          className="six wide column"
        >
          <h4
            className="ui inverted header"
          >
            About
          </h4>
          <div
            className="ui inverted link list"
            role="list"
          >
            <a
              className="item"
              href="mailto:hire@ghrecommender.io"
              onClick={[Function]}
              role="listitem"
            >
              <i
                aria-hidden="true"
                className="envelope outline icon"
              />
               Hire us
            </a>
            <a
              className="item"
              href="https://github.com/ghrecommender"
              onClick={[Function]}
              role="listitem"
            >
              <i
                aria-hidden="true"
                className="github icon"
              />
               GitHub
            </a>
            <a
              className="item"
              href="https://twitter.com/ghrecommender"
              onClick={[Function]}
              role="listitem"
            >
              <i
                aria-hidden="true"
                className="twitter icon"
              />
               Twitter
            </a>
          </div>
        </div>
        <div
          className="six wide column"
        >
          <p>
            Copyright Â© 2017 
            <a
              href="https://github.com/avli"
            >
              Andrey Lisin
            </a>
             and
             
            <a
              href="http://github.com/yurtaev"
            >
              Egor Yurtaev
            </a>
          </p>
          <p>
            The code is licensed under MIT.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
